### PR TITLE
[14.0][ADD] mrp_flattened_bom_xlsx_subcontracting_cost

### PR DESCRIPTION
**Pull request by BernatPForgeFlow**
Depends on https://github.com/OCA/manufacture-reporting/pull/99

### CODE REVIEW

File: mrp_flattened_bom_xlsx_subcontracting_cost/models/mrp_bom.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    subcontracting_cost = fields.Float(
+        string="Subcontracting Cost",
+        compute="_compute_subcontracting_cost",
+        digits=UNIT,
+    )
+
+    def _compute_subcontracting_cost(self):
+        supplier_info_obj = self.env["product.supplierinfo"]
+        for bom in self:
+            if bom.subcontractor_ids:
```
Comment (JordiBForgeFlow) [inline mrp_flattened_bom_xlsx_subcontracting_cost/models/mrp_bom.py]: @BernatPForgeFlow please also check if bom.type == 'subcontract'.

Comment (BernatPForgeFlow) [reply mrp_flattened_bom_xlsx_subcontracting_cost/models/mrp_bom.py]: Done!

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Comment by LoisRForgeFlow** [comment]
Dependency already merged.

/ocabot merge nobump

**Pull request merged**